src/PrestaShopBundle/Controller/Api/ApiController.php

```diff
@@ -197,16 +198,16 @@ protected function jsonResponse(
     /**
      * Checks if access is granted.
      *
-     * @param array $accessLevel
-     * @param string $controller name of the controller
+     * @param array $attributes
+     * @param string $subject name of the controller
      *
      * @return bool
      */
-    protected function isGranted(array $accessLevel, $controller)
+    protected function isGranted($attributes, $subject = null)
```

Comment:
Needed to comply with Symfony controller contract

Reply:
BC Break

Reply:
If I am not wrong, this change is backward compatible 🤔 ?

Reply:
Oops. You're right.